src/SqlFormatter.php

```diff
@@ -1187,7 +1127,7 @@ public function format(string $string, bool $highlight = true) : string
         foreach ($tokens as $i => $token) {
             // Get highlighted token if doing syntax highlighting
             if ($highlight) {
```

Comment:
i believe that all this `if ($highlight)` conditions can be replaced by having the highlighter passed as parameter here  instead of the constructor and having a `NoHighlightHighlighter` implementation.